src/universal/conf/testmode.conf

```diff
@@ -1,3 +1,5 @@
+include "app.conf"

```

Comment:
no from your PR but i noticed that in this file is missing `qa` rpc.apis option :P